scene/debugger/scene_debugger.cpp

```diff
@@ -1250,6 +1250,7 @@ void RuntimeNodeSelect::_setup(const Dictionary &p_settings) {
 	int pan_speed = p_settings.get("editors/panning/2d_editor_pan_speed", 20);
 	Array keys = p_settings.get("canvas_item_editor/pan_view", Array()).operator Array();
 	panner->setup(panning_scheme, DebuggerMarshalls::deserialize_key_shortcut(keys), simple_panning);
+	panner->set_viewport(root);
```

Comment:
If it's possible, that the panner is displayed not in the root window, but in a separate window, then this needs to be adjusted.

Reply:
I would say it's fine for now, as RuntimeNodeSelect only works for the root window.